scripts/setup_check.sh

```diff
@@ -21,8 +21,11 @@ set -euo pipefail
 export HOMEBREW_NO_ANALYTICS=1
 export HOMEBREW_NO_AUTO_UPDATE=1
 
-brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/c6f1cbd/Formula/clang-format.rb
-brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/c13eda8/Formula/swiftformat.rb
+brew install clang-format
```

Comment:
@paulb777  If we don't specify a particular version, then PRs will have to include formatting changes when a new version of formatter is released. It may be annoying and pollute PRs with irrelevant changes. I would prefer to keep a version number specifier to avoid this. 
WDYT?

Reply:
@maksymmalyhin I share the preference, but the old way to lock versions no longer works and I can't find another solution. If we can come up with a way, I'm happy to update the CI and instructions.

In the meantime, we need to unlock new contributors who don't have a way to install the old version - see the comment in the header.

Reply:
Ah, and you even explained it in the PR description, sorry for the silly comment.